docs/human-interface-guidelines.md

```diff
@@ -266,64 +266,64 @@ For more info, see the FreeDesktop.Org [menu entry](http://standards.freedesktop
 
 You may also include keywords in your launcher to help users find your app via search. These follow the convention of "X-GNOME-Keywords" (for in the app launcher) and "X-AppInstall-Keywords" (for in the app installer). For example, web browser might include "Internet" as a keyword even though it's not in the app's name, generic name, or description. As a result, a user searching for "Internet" will find the app. Here are some more examples:
 
-* Geary: **Email;Gmail**
+* Mail: **Email;Gmail**
 * Midori: **Internet;WWW;Explorer**
 * Files: **Folders;Browser;Explorer;Finder;Manager**
 * Terminal: **Command;Prompt;cmd;Emulator**
 * Scratch: **Notepad;IDE;Plain**
 * System Settings: **Control;Panel**
 * Shotwell: **Camera;Picture**
 
-Keywords should be single words in [title case](#title-case) and seperated with a semicolon.
+Keywords should be single words (in [title case](#title-case)) separated by semicolons.
 
 --------------------------------------
 
 See also: [Desktop Entry Specification](http://standards.freedesktop.org/desktop-entry-spec/latest/index.html) from FreeDesktop.org
 
 ## Contractor {#contractor}
 
-Contractor is a service and a protocol for exposing services easily between apps. It allows an app to interact with various other apps/services without hardcoding support for them. You simply add [contractor](https://launchpad.net/contractor) support, and then any service registered with contractor is now available for your app to use. Your app can integrate with contractor in two different ways:
+Contractor is a service and a protocol for exposing services easily between apps. It lets apps interact with other apps and services in lieu of hardcoded support. You simply add [Contractor](https://launchpad.net/contractor) support, and then any service registered with Contractor is now available for your app to use. Your app can integrate with Contractor in two different ways:
```

Comment:
I dislike "in lieu of". I think this is way outside the voice and the other wording was more clear.